hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -110,6 +114,15 @@ public void setBlockDeletionLimit(int limit) {
     this.blockLimitPerInterval = limit;
   }
 
+  @Config(key = "periodic.disk.check.interval.minutes",
+      defaultValue = "15",
+      type = ConfigType.LONG,
```

Comment:
Do we need a LONG? Can we just use an INT or TIME instead?

Reply:
@avijayanhwx I think it's because the method `scheduleWithFixedDelay` requires a LONG parameter, if we make the config item INT, then we may have some convertions (INT -> LONG).